collection/CheesiestMaster; Teyas's Guide to Aile.json

```diff
@@ -0,0 +1,801 @@
+        },
+        {
+            "name": "Swarm of Wasps (CR 30)",
+            "source": "TeyasGuidetoAile",
+            "page": 0,
+            "size": "L",
+            "type": {
+                "type": "beast",
+                "swarmSize": "T"
+            },
+            "alignment": [
+                "U"
+            ],
+            "ac": [
+                {
+                    "ac": 17,
+                    "from": [
+                        "natural armor"
+                    ]
+                }
+            ],
+            "hp": {
+                "formula": "20d8 + 200",
+                "average": 290
+            },
+            "speed": {
+                "walk": 5,
+                "fly": 30
+            },
+            "str": 11,
+            "dex": 21,
+            "con": 30,
+            "int": 1,
+            "wis": 7,
+            "cha": 1,
+            "senses": [
+                "blindsight 10 ft."
+            ],
+            "passive": 8,
+            "resist": [
+                "bludgeoning",
+                "piercing",
+                "slashing"
+            ],
+            "conditionImmune": [
+                "charmed",
+                "frightened",
+                "grappled",
+                "paralyzed",
+                "petrified",
+                "prone",
+                "restrained",
+                "stunned"
+            ],
+            "cr": "30",
+            "trait": [
+                {
+                    "name": "Swarm",
+                    "entries": [
+                        "The swarm can occupy another creature's space and vice versa, and the swarm can move through any opening large enough for a Tiny insect. The swarm can't regain hit points or gain temporary hit points."
+                    ]
+                }
+            ],
+            "action": [
+                {
+                    "name": "Bites",
+                    "entries": [
+                        "{@atk mw} {@hit 14} to hit, reach 0 ft., one target in the swarm's space. {@h}457 ({@damage 175d4}) piercing damage, or 215 ({@damage 86d4}) piercing damage if the swarm has half of its hit points or fewer."
+                    ]
+                }
+            ],
+            "environment": [
+                "desert",
+                "forest",
+                "grassland",
+                "hill",
+                "swamp",
+                "underdark",
+                "urban"
+            ],
+            "senseTags": [
+                "B"
+            ],
+            "damageTags": [
+                "P"
+            ],
+            "miscTags": [
+                "MW"
+            ],
+            "hasFluff": true,
+            "tokenUrl": "https://5etools-mirror-1.github.io/img/MM/Swarm of Wasps.png?v=1.147.13",
```

Comment:
drop the `?v=1.147.13` from the end--it isn't going to break anything, but it doesn't do anything either (besides maybe serving a stale version of the image from your cache)